tophat/yvm
# Highlight active Yarn version when listing remote versions

## Description
Highlight the version in use when listing remote versions. For #101 

## DevQA

### DevQA Steps
- Create a directory with a .yvmrc file 
- Run `yvm use` 
- Run `yvm ls-remote`
- See that the active version is highlighted in green
